leafmap/bokehmap.py

```diff
@@ -155,15 +161,15 @@ def add_cog_layer(
 
     def add_raster(
         self,
-        source,
-        band=None,
-        palette=None,
-        vmin=None,
-        vmax=None,
-        nodata=None,
-        attribution="",
+        source: str,
+        band: Optional[List] = None,
+        palette: Optional[str] = None,
+        vmin: Optional[float] = None,
+        vmax: Optional[float] = None,
+        nodata: Optional[float] = None,
+        attribution: Optional[str] = None,
```

Comment:
The default `attribution` value should be an empty string, not `None` because bokeh requires a string. The fail docs build is caused by this.